app/controllers/sessions_controller.rb

```diff
@@ -2,7 +2,12 @@ class SessionsController < ApplicationController
   skip_before_action :request_login
 
   def new
-    redirect_to "/auth/dfe"
+    if Rails.env.test?
```

Comment:
Suggestion:

```suggestion
    if Settings.features.signin_intercept
```

And add this to the settings. I'm also adding `signin_magic_link`, or something like that, to `Settings.features` too. Happy to approve as-is, though.